projects/js-packages/components/components/pricing-table/styles.module.scss

```diff
@@ -69,6 +73,10 @@
 			left: var( --border-offset, 0 );
 			right: var( --border-offset, 0 );
```

Comment:
These two lines (as well as the `--border-offset` variable declared earlier) probably don't make sense anymore right?